mayastor/src/bdev/nexus/nexus_child.rs

```diff
@@ -100,6 +100,33 @@ impl ToString for ChildState {
     }
 }
 
+/// Because the child states are kept separately from the rebuild states,
+/// when a rebuild is ongoing there are two states
+/// associated with the destination child:
+///     Rebuilding - from the rebuild job
+///     Faulted - from the child state
+/// Moac doesn't require fine grained state information from Mayastor
+/// The child states moac wants is:
+///     Online
+///     Faulted
+///     Rebuilding
+/// so here we inject a "rebuilding" state into the ChildState
+impl ChildState {
+    pub(crate) fn to_moac(self, rebuilding: bool) -> String {
```

Comment:
Should we explicitly use the term "moac"? Maybe we should make the naming more generic in case moac is switched out for something else. Maybe something like "to_control_plane_state" or "to_external_state"?

Reply:
yep we shouldn't... hmm to_rpc_state?

Reply:
to_public